internal/statemachine/classic_test.go

```diff
@@ -1219,7 +1220,9 @@ func TestCreateChroot(t *testing.T) {
 		stateMachine.ImageDef = ImageDefinition{
 			Architecture: getHostArch(),
 			Series:       getHostSuite(),
-			Rootfs:       &RootfsType{},
+			Rootfs: &RootfsType{
+				Pocket: "proposed",
```

Comment:
Should we maybe add a test that makes sure the right pocket combinations are enabled for the selected pocket types? Like, to make sure if someone selects `security`, they won't get updates or proposed by accident.

Reply:
So when I was originally thinking of how to test this I came up with 2 ways to do it, both of which have some drawbacks:

1. Run the whole `createChroot` function for each pocket and check the contents of `/etc/apt/sources.list` in the chroot. This has the drawback of needing to run `debootstrap` 4 times, which would make the test suite even heftier than it already is.

2. Just unit test the `generatePocketList` function. This is trivial and won't make the tests noticeably slower. However, since `generatePocketList` essentially just returns a pre-defined variable it seemed like a test that was so trivial it wouldn't really matter.

Thoughts on implementing either of these tests?